lib/inferno/db/schema.rb

```diff
@@ -62,6 +62,9 @@
       
       primary_key [:id]
       
+      index [:test_session_id, :test_group_id], :name=>:results_session_group
```

Comment:
These need to be added via a migration.

Reply:
Hmmm, I'm not sure when / why I added these.  It certainly is out of scope of this ticket, and I wonder if it was something experimental that I was working on back when we were analyzing indices.  I'll remove these changes.

Reply:
Reverted.